### PR TITLE
Rebased to latest upstream

### PR DESCRIPTION
**Pull request by tilmankamp**
@kdavis-mozilla : Builds for me on base of newest/rebased smarthome. Please ignore the Travis result for this, as the smarthome changes have to be merged and built first.

### CODE REVIEW

**Comment by kdavis-mozilla** [comment]
@Cwiiis Can you take a look at this?

**Comment by tilmankamp** [comment]
@Cwiiis : Please review

File: bundles/org.openhab.ui.paperui/patch/web/js/constants.js [hunk 1]
```diff
@@ -13,6 +13,5 @@ angular.module('PaperUI.constants', [])
   'configuration': true,
   'extensions': true,
   'rules': true,
-  'preferences': true,
-  'groups': false
```
Comment (Cwiiis) [inline bundles/org.openhab.ui.paperui/patch/web/js/constants.js:17]: Is the removal of 'groups' intentional?

**Comment by Cwiiis** [comment]
Looks good to me, left one comment. I assume it still looks ok when we run it, if anything's off we can fix it later.

File: bundles/org.openhab.ui.paperui/patch/web/js/constants.js (hunk 1 above)
Comment (tilmankamp) [inline bundles/org.openhab.ui.paperui/patch/web/js/constants.js:17]: It was done by this commit of the source repo:
https://github.com/mozilla/openhab2-addons/commit/7a041cb7bb72238e3b3b7fefa1b238cd69a3b613

Comment (Cwiiis) [inline bundles/org.openhab.ui.paperui/patch/web/js/constants.js:17]: That link doesn't seem to be right, but if it's not in the source, then sure.

**Comment by kdavis-mozilla** [comment]
@tilmankamp Are you going to merge?

**Pull request merged**